baseplate/diagnostics/metrics.py

```diff
@@ -39,18 +39,15 @@ def __init__(self, batch, name):
     def on_start(self):
         self.timer.start()
 
-    def on_annotate(self, key, value):  # pragma: nocover
```

Comment:
what this originally unneeded, since it was duplicating `SpanObserver.on_annotate`?